File: Meta/2024/PracticeRound/CPP/linebyline.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>
#include <fstream>
using namespace std;

int main() {
    int length;
    ofstream cout("output.txt");
    cout.precision(10);
    cin >> length;
    for(int i = 1; i <= length; i++) {
        long double N, P;
        cin >> N >> P;
        //
        long double Prob;
        Prob = P * (pow((100.0/P), 1.0/N) - 1.0);
        //
        cout << "Case #" << i << ": " << Prob << endl;
    }
}

File: Meta/2024/PracticeRound/Python/LinebyLine.py
import sys
from math import pow
i = 0
for line in sys.stdin:
    if line == "\n":
        break
    if i == 0:
        no_test = line
    else:
        N, P = list(map(float, line.split()))
        Prob = P * (pow((100/P), 1/N) - 1)
        with open("Meta\\2024\\PracticeRound\\output.txt", "a") as f:
            f.write(f"Case #{i}: {Prob}\n")
    i += 1